performance.html

```diff
@@ -25,6 +25,14 @@ <h1 class="display-4">Dex continuous benchmarking</h1>
       regression: 'Regression example',
       fluidsim: 'Fluid simulation example',
       fused_sum: 'Fused summation micro-benchmark',
+      gaussian: 'Generating 1,000,000 Gaussian random variates micro-benchmark',
+      jvp_matmul 'Matrix multiply as generated by forward AD (500x500 times 500x500) micro-benchmark',
+      matmul_big: 'Matrix multiply (500x500 times 500x500) micro-benchmark',
+      matmul_small: 'Matrix multiply (1000-batch of 10x10 times 10x10) micro-benchmark',
+      matvec_big: 'Matrix-vector multiply (10000x10000 times 10000) micro-benchmark',
+      matvec_small: 'Matrix-vector multiply (10000-batch of 10x10 times 10) micro-benchmark',
```

Comment:
I feel like we should have unbatched benchmarks too, since that's an even more common case.